packages/create-rspack/index.js

```diff
@@ -69,17 +70,17 @@ yargs(hideBin(process.argv))
 
 		fs.mkdirSync(root, { recursive: true });
 		const srcFolder = path.resolve(__dirname, `template-${template}`);
-		copyFolder(srcFolder, targetDir);
+		copyFolder(srcFolder, root, targetDir);
 		const pkgManager = getPkgManager();
 		console.log("\nDone. Now run:\n");
-		console.log(`cd ${targetDir}\n`);
+		console.log(`cd ${getProjectDir(targetDir)}\n`);
```

Comment:
`getProjectDir(targetDir)` is called multiple times, can we use a variable to store it

Reply:
done